### ISSUE TITLE
SlowConsumerError with PushSubscription and sub.next_msg()

### ISSUE THREAD

**Issue by KrumBoychev** [nats-io/nats.py#372]
The following code generate SlowConsumerError even the messages are processed by the consumer:

```python
import asyncio
import nats
from nats.errors import TimeoutError


async def main():
    errors = []

    async def error_cb(err):
        errors.append(err)

    nc = await nats.connect(error_cb=error_cb)
    js = nc.jetstream()

    await js.add_stream(name="cqsub", subjects=["quux"])
    sub = await js.subscribe("quux", durable="can", pending_bytes_limit=100)
    for i in range(5):
        await js.publish("quux", f'Hello World {i}'.encode())

    while True:
        try:
            msg = await sub.next_msg()
            print(f"{msg.data} len: {len(msg.data)}")
            await msg.ack()
        except TimeoutError:
            break

    for i in range(5):
        await js.publish("quux", f'Hello World {i}'.encode())

    print(errors)
    breakpoint()

    await nc.close()


if __name__ == '__main__':
    asyncio.run(main())
```

```
 bin/test_js_limits.py 
b'Hello World 0' len: 13
b'Hello World 1' len: 13
b'Hello World 2' len: 13
b'Hello World 3' len: 13
b'Hello World 4' len: 13
[SlowConsumerError(), SlowConsumerError(), SlowConsumerError()]
> /home/bebop/Projects/GIT/bwnormd/bin/test_js_limits.py(36)main()
     35 
---> 36     await nc.close()
     37 

ipdb> sub.pending_bytes
-65
ipdb> sub._sub.pending_bytes
91
ipdb> sub.pending_msgs
2
ipdb> 

```

It looks that next_msg is not subtracting from the correct pending_bytes.

https://github.com/nats-io/nats.py/blob/main/nats/js/client.py#L629

https://github.com/nats-io/nats.py/blob/main/nats/aio/subscription.py#L163

https://github.com/nats-io/nats.py/blob/main/nats/aio/client.py#L1722

**Comment by wojtekbacza-coco** [comment]
Any update on it? It makes push subscribers totally unusable, as there is a hard limit of bytes after which they break.

**Comment by wallyqs** [comment]
Fixed in v2.1.5: https://github.com/nats-io/nats.py/releases/tag/v2.1.5
